homeassistant/components/tuya/switch.py

```diff
@@ -888,6 +901,32 @@ def __init__(
         self.entity_description = description
         self._attr_unique_id = f"{super().unique_id}{description.key}"
 
+    async def async_added_to_hass(self) -> None:
+        """Called when entity is added to Home Assistant."""
+        await super().async_added_to_hass()
```

Comment:
Please look at `ring` integration (`async_check_create_deprecated` function) to see how issues should be created and deleted for deprecated entities.
<https://github.com/home-assistant/core/blob/5930ac6425e06cb9097f7b58a927fb7e6a972186/homeassistant/components/ring/binary_sensor.py#L81>